### PR TITLE
V3 - feat: make ResultBlock responsive and colorable

### PR DESCRIPTION
**Pull request by trulshj**

### CODE REVIEW

File: src/components/customerCases/customerCase/sections/results/resultsBlock.module.css [hunk 1]
```diff
@@ -93,6 +94,28 @@
   border-top-left-radius: 0;
 }
 
-.highlightRow .highlightCard {
-  width: 100%;
+@media (max-width: 640px) {
```
Comment (idamand) [inline src/components/customerCases/customerCase/sections/results/resultsBlock.module.css]: Any reason for breaking at 640px? We have defined breaking points at 834 for tablet and 425 for mobile, so just wondering why 640px was used here :)

Comment (trulshj) [reply src/components/customerCases/customerCase/sections/results/resultsBlock.module.css]: just a derp, saw 640 being used somewhere 
image

**Review by idamand — APPROVED** [review]
Looks good :)

File: src/components/customerCases/customerCase/sections/results/ResultsBlock.tsx [hunk 2]
```diff
@@ -5,23 +5,34 @@ import styles from "./resultsBlock.module.css";
...
+}: ResultsBlockProps) {
   return (
     section.resultsBlockTitle && (
       <div className={styles.wrapper}>
         <div>
-          <StackedHighlights section={section}></StackedHighlights>
+          <StackedHighlights
+            section={section}
+            blockColor={blockColor}
+          ></StackedHighlights>
```
Comment (petterhh) [inline src/components/customerCases/customerCase/sections/results/ResultsBlock.tsx]:
```suggestion
          <StackedHighlights
            section={section}
            blockColor={blockColor}
          />
```
I think you can simplify like this 😊

**Pull request merged**